torch/_subclasses/fake_tensor.py

```diff
@@ -702,7 +702,8 @@ def tolist(self):
         for _ in range(self.shape[0]):
             s = shape_env.create_unbacked_symint()
             # max value?
-            torch._constrain_as_size(s, min=2)
+            torch._check_is_size(s)
```

Comment:
Is this safe? Is the `>= 2` needed below?

Reply:
I wonder if the >= 2 was added due to the 0/1 specialization issue. Also not sure if the 0/1 specialization issue has been fixed for this case. @ezyang

Reply:
I think `_check_is_size` temporarily assumes `>= 2` whereas the check below will always force it. So I'm leaving in the check, don't want to change any behavior in this test right now.